src/mywxtools.h

```diff
@@ -179,3 +179,14 @@ static wxBitmap ConvertBufferToWxBitmap(vector<uint8_t> &buf, wxBitmapType bmt)
     wxBitmap bm(im, 32);
     return bm;
 }
+
+inline static bool IsWordSep(wxChar ch) {

```

Comment:
Are these used in multiple files? If they are only used in the selection code, it be best to move them close to where they are used

Reply:
Hi :-) Thanks for your feedback. It looks like the callers are only in `selection.h` itself, so I updated the PR/commit.